predicts_api.py

```diff
@@ -18,18 +17,16 @@ def start():
 def predict():
     if request.method == 'POST':
         try:
-            json_path = request.get_data()
-            with open(json_path, "r") as f:
-                json_data = json.load(f)
-            df = pd.DataFrame({0: json_data}).transpose()
+            json_data = request.get_json()
+            if json_data is None:
+                return jsonify("Expected json")
+            else:
+                df = pd.DataFrame({0: json_data}).transpose()
```

Comment:
Model predict method works with array-like input. Is there any reason to use pandas.DataFrame?

Reply:
done